src/components/views/LocalModList.vue

```diff
@@ -36,10 +36,22 @@
                 </div>
             </div>
             <div class="tabs">
-                <ul>
-                    <li :class="[{'is-active': activeTab === 0}]" @click="changeTab(0)"><a>All ({{ getTabCountForAll }})</a></li>
-                    <li :class="[{'is-active': activeTab === 1}]" @click="changeTab(1)"><a>Tracked ({{ getTabCountForInstalled }})</a></li>
-                    <li :class="[{'is-active': activeTab === 2}]" @click="changeTab(2)"><a>Dependencies ({{ getTabCountForDependencies }})</a></li>
+                <ul class="margin-bottom">
+                    <li :class="[{'is-active': activeTab === 0}]" @click="changeTab(0)">
```

Comment:
IMO `:class` and `@click` should use LocalModTabs enums instead of magic numbers.

Reply:
We can't reference enums directly in here so the only alternative would be a method per option

Reply:
If we define something like this in the component:
```
        private activeTab: LocalModTabs = LocalModTabs.INSTALLED;
        private tabs = LocalModTabs;
```

Then the enum options can be accessed in the template via `tabs`. Not sure if this is bad practice for some reason?